File: Código_JS/parImpar.js
//Determinar si un numero es par o impar


const prompt = require('prompt-sync')();

const numero = parseInt(prompt("Ingresa un número ->"));

if (numero % 2 === 0) {
  console.log("El número es par.");
}else { 
  console.log("El número es impar.");
}

File: Códigos_Java/jorge/ParImpar.java
package Valor_Absoluto.Valor_Absoluto.src;
import javax.swing.JOptionPane;
public class ParImpar {
    static int n;
    public static void main(String[] args) throws Exception {
        n=Integer.parseInt(JOptionPane.showInputDialog("Ingresa un numero y te diré si es\nPar o Impar;\t") );
        JOptionPane.showMessageDialog(null,"El  Numero "+n+" es; "+(((n%2)== 0) ? ("par") : ("impar")));
    }
}
